.github/workflows/ci.yaml

```diff
@@ -112,6 +112,20 @@ jobs:
       - name: Test
         run: make test-code
 
+  build-binaries:
```

Comment:
Why image build does not fail?

Reply:
You mean “There are several other places having same issue”, so the make build should have failed?…

Reply:
He meant, there are many places you see int64, fix all those.

Reply:
And curious why image build didn't fail, it also has binary build included.

Reply:
> And curious why image build didn't fail, it also has binary build included.

could be in release.yaml it's using `ubuntu-20.04` while here we are using `ubuntu-latest`?...